Tests/AlgoliaSearch/ReindexCommandTest.php

```diff
@@ -83,7 +86,7 @@ public function reIndex($use_entity_alias, $safe = true)
 
         $input = new ArrayInput($options);
 
-        $this->command->run($input, new ConsoleOutput());
+        $this->command->run($input, new NullOutput());
```

Comment:
Travis fails and I get the same result locally. I manage to get the tests passing but using `ConsoleOutput` like it was before.
Why did you change this to `NullOutput`?

After that I believe we'll be ready to merge! 🎉